### PR TITLE
Upgraded to version 4.1.0

### PR DESCRIPTION
**Pull request by luigisaggese**

### CODE REVIEW

**Comment by deRonbrown** [comment]
This is great. Thanks for the work, @luigisaggese.

We just released v4.1.0 for iOS that includes compatibility with iOS 10 and the `UserNotifications` framework. Do you mind updating to that version and including the additional public API methods?

Additional `Localytics` class methods:

```
+ (void)didReceiveNotificationResponseWithUserInfo:(nonnull NSDictionary *)userInfo;
+ (void)didRegisterUserNotificationSettings:(nonnull UIUserNotificationSettings *)notificationSettings;
+ (void)didRequestUserNotificationAuthorizationWithOptions:(NSUInteger)options granted:(BOOL)granted;
```

Additional `LLMessagingDelegate` method:

```
- (nonnull UNMutableNotificationContent *)localyticsWillDisplayNotificationContent:(nonnull UNMutableNotificationContent *)notification forPlacesCampaign:(nonnull LLPlacesCampaign *)campaign;
```

File: LocalyticsXamarin/LocalyticsXamarin.iOS/StructsAndEnums.cs [hunk 1]
```diff
@@ -16,4 +16,12 @@ public enum LLProfileScope : long
 		Application,
 		Organization
 	}
+
+	[Native]
+	public enum LLRegionEvent : long
+	{
+		nter,
+		xit
```
Comment (deRonbrown) [inline LocalyticsXamarin/LocalyticsXamarin.iOS/StructsAndEnums.cs]: Looks like the first `E` characters got cut off there.

**Comment by luigisaggese** [comment]
Please check pull request update to last release 4.1.0

**Comment by deRonbrown** [comment]
@luigisaggese, thanks for the update, but we've made a few changes to the core SDKs for better support for v4 running on Xamarin. PR #4 is the official 4.1 release. Please update your app to pull from this branch.

Your original iOS contributions have not gone unnoticed. As you can see in that PR, we pulled in your 4.0.1 commits. Thanks and let us know if you have any questions or find any issue.

**Pull request closed without merging**